jscomp/build_tests/zerocycle/src/bar.res

```diff
@@ -0,0 +1,14 @@
+// one-file false positive
+
```

Comment:
Can you add a comment with a link to the issue? So it's clear what this is a false positive for.

Reply:
done 👍